src/client/gameui.cpp

```diff
@@ -129,9 +129,9 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< "pos: (" << (player_position.X / BS)
 			<< ", " << (player_position.Y / BS)
 			<< ", " << (player_position.Z / BS)
-			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "° "
+			<< ") | yaw: " << (wrapDegrees_0_360(cam.camera_yaw)) << "\xC2\xB0 "
```

Comment:
isn't this utf-8 specific?
shouldn't this be an `\uabcd` style escape?

Reply:
This is exactly equivalent to what is in the original code. The original code contains the utf-8 character spelled out directly, I just converted that into C escapes to bring it down to ASCII. "\uabcd" is a "wide char string" which would prompt for much more intrusive code change, at least here and I am not sure whether elsewhere too.